src/DataFixtures/ORM/LoadEventData.php

```diff
@@ -547,6 +547,7 @@ public function getDependencies()
         return [
             LoadAdherentData::class,
```

Comment:
`LoadAdherentData` is no longer required here, because `LoadCommitteeData` depends on `LoadAdherentData`